backend/app/testbot/llm.py

```
@@ -0,0 +1,32 @@
+
+def ask_llm(prompt):
+    """
+    Given a prompt, call the Gemini API and return the generated answer.
+    """
+    payload = {
+        "contents": [
+            {"parts": [{"text": prompt}]}
+        ]
+    }
+    headers = {"Content-Type": "application/json"}
+    response = requests.post(GEMINI_API_URL, headers=headers, json=payload)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)